nestor_api/utils/dict.py

```diff
@@ -15,8 +15,7 @@ def deep_merge(destination: dict, source: dict) -> dict:
 
     def _deep_merge_rec(dest, src):
         for key in src:
-            # pylint: disable=bad-continuation
-            if key in dest and isinstance(dest.get(key), dict) and isinstance(src[key], dict):
+            if isinstance(dest.get(key), dict) and isinstance(src[key], dict):
```

Comment:
Why do we access 2 dicts with 2 different ways : `[]` & `get`

Reply:
Since we are iterating over `src`, the key can be missing in `dest` and in this case the `[]` operator will throw whereas the `.get()` will return `None`.
In this case I removed the first check which was redundant.